### PR TITLE
Add Support for Julia v0.7/v1.0

### PR DESCRIPTION
**Pull request by ccoffrin**
Fairly strait forward.  It took me about an hour to figure out that sparse matrix transpose needed to be wrapped in a `copy`.

CC @chriscoey

### CODE REVIEW

**Comment by chriscoey** [comment]
great minds submit PRs at exactly the same time

**Comment by ccoffrin** [comment]
Ha ha ya.  Best to merge yours first, then I can make any needed updates to new v0.6-only code.

**Comment by chriscoey** [comment]
the readme here can be cleaned up too

File: src/ConicNonlinearBridge.jl [hunk 1]
```diff
@@ -4,6 +4,20 @@ module ConicNonlinearBridge
 using MathProgBase
 using JuMP
 
+using Compat.LinearAlgebra
+using Compat.SparseArrays
+
+if VERSION < v"0.7.0-"
```
Comment (mlubin) [inline src/ConicNonlinearBridge.jl]: Why the empty block?

File: test/runtests.jl [hunk 2]
```diff
@@ -44,13 +44,13 @@ end
...
     @test issubset([:Free, :Zero, :NonNeg, :NonPos, :SOC, :SOCRotated, :ExpPrimal], MathProgBase.supportedcones(solver))
     m = MathProgBase.ConicModel(solver)
-    MathProgBase.loadproblem!(m, [0, -2, -1], [1 0 0; -eye(3)], [1, 0, 0, 0], [(:Zero, 1), (:SOC, 2:4)], [(:Free, 1:3)])
+    MathProgBase.loadproblem!(m, [0, -2, -1], [1 0 0; Matrix(-1.0I, 3, 3)], [1, 0, 0, 0], [(:Zero, 1), (:SOC, 2:4)], [(:Free, 1:3)])
     MathProgBase.setvartype!(m, [:Cont, :Bin, :Int])
     MathProgBase.setwarmstart!(m, [3, 4, 2])
     @test m.nlp_model.colCat[1:3] == [:Cont, :Bin, :Int]
     @test m.solution[1:3] == [3, 4, 2]
     @test MathProgBase.numvar(m) == 3
-    @show MathProgBase.numconstr(m) == 4
```
Comment (chriscoey) [inline test/runtests.jl:38]: oops

File: src/ConicNonlinearBridge.jl (hunk 1 above)
Comment (ccoffrin) [reply src/ConicNonlinearBridge.jl]: No good reason.  Removed.

**Comment by ccoffrin** [comment]
@chriscoey I don't know how to update the readme becouse I don't know how this package is used in Pavito and other MPB packages.  Can you take a stab at that one?

**Comment by chriscoey** [comment]
ok see #14

**Pull request merged**